src/test/migrations/reset.ts

```diff
@@ -58,17 +58,11 @@ const resetMigrationSteps = async () => {
     }
 
     // === 3. Create reset step
-    const { filePath, fileName } = await createMigrationStep('reset')
+    const { fileName } = await createMigrationStep('reset')
     // eslint-disable-next-line @typescript-eslint/no-var-requires,global-require,import/no-dynamic-require
-    const resetStep = require(filePath).default
 
-    await DB.tx(async (t) => {
-      // === 4. Run reset step
-      await resetStep(t, fileName)
```

Comment:
autorun removed @minotogna

Reply:
meaning that `migration-steps:reset` will only delete the files and create the reset step

afterwards it is expected to align the DB with `migration-steps:run`